### PR TITLE
Prevent check and jmx command from depending on config Set method

### PR DESCRIPTION
**Pull request by ajacquemot**
### What does this PR do?

Get rid of config Set methods in `cmd` package.

### Motivation

Clean up and prepare core agent config refactoring

### Additional Notes

Needed for refactoring the core-agent config

### CODE REVIEW

File: cmd/agent/android/main_android.go [hunk 1]
```diff
@@ -10,6 +10,7 @@ package ddandroid
 import (
 	"fmt"
 	"log"
+	"strings"
```
Comment (mfpierre) [inline cmd/agent/android/main_android.go]: How was it working before? 🤔

**Review by mfpierre — APPROVED** [review]
LGTM @arbll for a second look?

File: cmd/agent/android/main_android.go (hunk 1 above)
Comment (ajacquemot) [reply cmd/agent/android/main_android.go]: good question

**Review by gmmeyer — APPROVED** [review]
🎉

**Pull request merged**